spec/oncocore_genetics_vs.txt

```diff
@@ -60,6 +61,50 @@ LNC#LA18196-8   "Likely fetal"    // SCT#703065002:2931005 "Probably fetal origi
 LNC#LA18197-6   "Unknown genomic origin"   //SCT#261665006 "Unknown"
 LNC#LA26807-0   "De novo"         //  SCT#7147002         "New (de novo)"
 
+ValueSet:       GeneticSpecimenTypeVS
+Description:    "The type of specimen analyzed in a genetic test."
+SPTY#AMN       "Amniotic fluid"
+SPTY#BIFL       "Bile Fluid"
+SPTY#BLD       "Whole blood"
+SPTY#BLDA       "Blood arterial"
+SPTY#BLDCO       "Cord blood"
+SPTY#BLDV       "Blood venous"
+SPTY#BON       "Bone"
+SPTY#BX       "Biopsy"
```

Comment:
Remove biopsy, as it refers to the method through which the specimen was obtained, but not the tissue/fluid that the specimen is made up of.

Reply:
Exactly. Done.